anvil/observables.py

```diff
@@ -4,17 +4,23 @@
 observables.py
 """
 import numpy as np
-from scipy.signal import correlate
-from math import ceil, pi, sin
+import scipy.signal, scipy.optimize
+import scipy.optimize as optim
```

Comment:
please fix this: two separate lines, no alias, remove duplication

also fix the ordering of all the imports